src/main/java/com/nekomaster1000/infernalexp/entities/VolineEntity.java

```diff
@@ -1,30 +1,84 @@
 import net.minecraft.entity.ai.attributes.AttributeModifierMap;
 import net.minecraft.entity.ai.attributes.Attributes;
-import net.minecraft.entity.ai.goal.*;
+import net.minecraft.entity.ai.goal.AvoidEntityGoal;
+import net.minecraft.entity.ai.goal.Goal;
+import net.minecraft.entity.ai.goal.HurtByTargetGoal;
+import net.minecraft.entity.ai.goal.LookAtGoal;
+import net.minecraft.entity.ai.goal.LookRandomlyGoal;
+import net.minecraft.entity.ai.goal.MeleeAttackGoal;
+import net.minecraft.entity.ai.goal.NearestAttackableTargetGoal;
+import net.minecraft.entity.ai.goal.PanicGoal;
+import net.minecraft.entity.ai.goal.WaterAvoidingRandomWalkingGoal;
 import net.minecraft.entity.monster.MonsterEntity;
 import net.minecraft.entity.monster.piglin.AbstractPiglinEntity;
 import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
 import net.minecraft.item.Items;
-import net.minecraft.item.crafting.Ingredient;
+import net.minecraft.nbt.CompoundNBT;
+import net.minecraft.network.datasync.DataParameter;
+import net.minecraft.network.datasync.DataSerializers;
+import net.minecraft.network.datasync.EntityDataManager;
+import net.minecraft.particles.ParticleTypes;
 import net.minecraft.potion.Effects;
 import net.minecraft.util.DamageSource;
 import net.minecraft.util.SoundEvent;
 import net.minecraft.util.SoundEvents;
 import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.DifficultyInstance;
+import net.minecraft.world.IServerWorld;
 import net.minecraft.world.World;
+import net.minecraftforge.api.distmarker.Dist;
+import net.minecraftforge.api.distmarker.OnlyIn;
+
+import javax.annotation.Nullable;
+import java.util.HashMap;
+import java.util.Map;
 
 public class VolineEntity extends MonsterEntity {
 
-    public static final Ingredient TEMPTATION_ITEMS = Ingredient.fromItems (Items.MAGMA_CREAM);
+//    public static final List<Item> EAT_ITEMS = new ArrayList<>(
+//            Arrays.asList(Items.MAGMA_CREAM, Items.GOLD_INGOT, Items.GOLD_BLOCK, Items.GOLD_ORE, Items.NETHER_GOLD_ORE, Items.GOLDEN_AXE, Items.GOLDEN_PICKAXE,
+//                    Items.GOLDEN_HOE, Items.GOLDEN_SWORD, Items.GOLDEN_SHOVEL, Items.GOLDEN_HELMET, Items.GOLDEN_CHESTPLATE, Items.GOLDEN_LEGGINGS, Items.GOLDEN_BOOTS)
+//    );
+
+    public static final Map<Item, Integer> EAT_ITEMS = new HashMap<Item, Integer>() {{
+        put(Items.MAGMA_CREAM, 0);
+        put(Items.GOLD_NUGGET, 0);
+        put(Items.GOLD_INGOT, 4);
+        put(Items.GOLD_BLOCK, 22);
+        put(Items.GOLD_ORE, 4);
+        put(Items.NETHER_GOLD_ORE, 4);
+        put(Items.GOLDEN_AXE, 9);
+        put(Items.GOLDEN_PICKAXE, 9);
+        put(Items.GOLDEN_SWORD, 5);
+        put(Items.GOLDEN_HOE, 5);
+        put(Items.GOLDEN_SHOVEL, 2);
+        put(Items.GOLDEN_HELMET, 11);
+        put(Items.GOLDEN_CHESTPLATE, 20);
+        put(Items.GOLDEN_LEGGINGS, 18);
+        put(Items.GOLDEN_BOOTS, 13);
+    }};
```

Comment:
This should be converted into a tag or recipe type for modpacks to customize.

Reply:
Is there anyway with tags to hold some kind of value?

Reply:
A loot table then, kinda like piglin bartering?
See: https://mcasset.cloud/1.16.5/data/minecraft/loot_tables/gameplay/piglin_bartering.json